docs/user-guide/api-mediation/api-mediation-caching-service.md

```diff
@@ -1,13 +1,14 @@
 # Using the Caching Service
 
-As an API developer, you can use the Caching Service as a storage solution to enable resource sharing between service instances, thereby ensuring High Availability of services. The Caching Service makes it possible to store, retrieve, and delete data associated with keys. The Caching Service is designed to make resource sharing possible for services that cannot be made stateless in two ways:
+As an API developer, you can use the Caching Service as a storage solution to enable resource sharing between service instances, thereby ensuring High Availability of services. The Caching Service makes it possible to store, retrieve, and delete data associated with keys. The Caching Service is designed to make resource sharing possible for services that cannot be made stateless by using following backends:
 
-- Using VSAM to store key/value pairs for production
-
-- Using InMemory
+- Using Infinispan that is part of Caching Service
+- Using Redis running off-platform
+- {Deprecated} Using VSAM
+- {Development Use Only} Using InMemory
```

Comment:
In Docusaurus V3, curly braces need to be escaped:
```suggestion
- \{Deprecated\} Using VSAM
- \{Development Use Only\} Using InMemory
```

Escaping them should fix the build error 🙏 
![image](https://github.com/user-attachments/assets/0b152517-0b4d-4308-ae97-dd4b6edf5216)